jdbc-h2/src/main/java/io/kestra/runner/h2/H2Functions.java

```diff
@@ -11,20 +11,17 @@
 import net.thisptr.jackson.jq.Versions;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
 public class H2Functions {
-    private static final Scope rootScope;
     private static final Scope scope;
 
     static {
-        rootScope = Scope.newEmptyScope();
-        BuiltinFunctionLoader.getInstance().loadFunctions(Versions.JQ_1_6, rootScope);
         scope = Scope.newEmptyScope();
```

Comment:
While you're at it, better to init the scope at its declaration site instead of inside this static block. It's clearer and avoid possible misuse and races.